tools/apigee-x-trial-provision/pipeline.sh

```diff
@@ -1,17 +1,19 @@
 #!/bin/sh
 
 # Copyright 2020 Google LLC
-# 
+#
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
-# 
+#
 # <http://www.apache.org/licenses/LICENSE-2.0>
-# 
+#
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-echo "no pipeline required"
+gcloud builds submit --config ./cloudbuild.yaml \
```

Comment:
is this synchronous and returns the correct status code? or should we log poll the result? pipeline.sh should return non zero if the pipeline fails

Reply:
synchronous. fails if the secondary pipeline fails.